README.md

```diff
@@ -21,7 +21,97 @@ Use robust linting like we do
 
 ## Contributing
 
-It uses [lerna](https://lernajs.io) to maintain all packages within one repository, so take a look at its features. 
+It uses [lerna](https://lernajs.io) to maintain all packages within one repository, so take a look at its features.
+
+## How to install
+
+### Linter for React projects `eslint-config-datarockets`
+
+[![npm version](https://img.shields.io/npm/v/eslint-config-datarockets.svg?style=flat-square)](https://www.npmjs.com/package/eslint-config-datarockets)
+
+Sharable eslint config for datarocket's projects.
+
+It's based on [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript) with some specific changes.
+
+1. Installing deps:
+
+  With `npm`:
+
+  ```
+  (
+    export PKG=eslint-config-datarockets;
+    npm info "$PKG@latest" peerDependencies --json | command sed 's/[\{\},]//g ; s/: /@/g' | xargs npm install --save-dev "$PKG@latest"
+  )
```

Comment:
What if we make those installation commands oneliners:
```
export PKG=eslint-config-datarockets; npm info "$PKG@latest" peerDependencies --json | command sed 's/[\{\},]//g ; s/: /@/g' | xargs npm install --save-dev "$PKG@latest"
```

Will it work?

Reply:
You know, I'd prefer the version with more break lines to show more clearly what this command does.
Something like this:
```
(
  export PKG=eslint-config-datarockets;
  npm info "$PKG@latest" peerDependencies --json \
  | command sed 's/[\{\},]//g ; s/: /@/g' \
  | xargs npm install --save-dev "$PKG@latest"
)
```
How do you feel it?